src/bun.js/api/server.zig

```diff
@@ -6165,6 +6173,11 @@ pub fn NewServer(comptime NamespaceType: type, comptime ssl_enabled_: bool, comp
             defer {
                 // uWS request will not live longer than this function
                 request_object.request_context = JSC.API.AnyRequestContext.Null;
+                // JS signal is lazy created when needed and we need to keep it alive so we can call the events
+                const js_signal = RequestPrototype__signalGetCachedValue(request_value);
```

Comment:
this will crash if there's a pending exception

```suggestion
                const js_signal = if (!globalThis.hasPendingException()) RequestPrototype__signalGetCachedValue(request_value) else JSValue.empty;
```

Reply:
fixed here: https://github.com/oven-sh/bun/pull/11672/commits/3732a78555ae6780871cdf6ff6b23ddf89c49adb

Reply:
Handled here:
https://github.com/oven-sh/bun/pull/11672/files#diff-e9484fda309708f80db81a125281dd0b9fdadaf3d2400ae8e78a2b8fc14d85ebR6173-R6180